app/controllers/projects_controller.rb

```diff
@@ -5,5 +5,24 @@ def index
     end
     def show 
         @project = Project.find(params[:id])
+
+        #make last name array accessible to form for user addition from UI
+        @user_last_names = ["<Select User>"]
+        User.all.each do |user|
```

Comment:
This could become really slow (and just kill your app) if you have millions of users